docs/features/SupportedRFCs.md

```diff
@@ -105,14 +105,14 @@ provides notes about the implementation of specific RFCs.
 
 ### AIP 2.0
 
-All RFCs listed in [AIP 2.0](https://github.com/hyperledger/aries-rfcs/tree/main/concepts/0302-aries-interop-profile#aries-interop-profile-version-20) (including the sub-targets) 
+All RFCs listed in [AIP 2.0](https://github.com/hyperledger/aries-rfcs/tree/1.0.0-rc2/concepts/0302-aries-interop-profile#aries-interop-profile-version-20) (including the sub-targets) 
 are fully supported in ACA-Py **EXCEPT** as noted in the table below.
 
 | RFC | Supported | Notes |
  --- | :--: | -- |
 | [0023-did-exchange](https://github.com/hyperledger/aries-rfcs/tree/b3a3942ef052039e73cd23d847f42947f8287da2/features/0023-did-exchange)   | :warning:   |   Not using DIDDoc conventions yet, still using DID format of 0160-connections (which is incorrect and outdated). Also using incorrect format for `did:peer`  (or not using a `did:` prefix at all) |
 | [0211-route-coordination](https://github.com/hyperledger/aries-rfcs/tree/b3a3942ef052039e73cd23d847f42947f8287da2/features/0211-route-coordination)   | :warning:  | Only pre-AIP 2.0 version. Must be updated to use `did:key` for full AIP 2.0 support  |
-| [0317-please-ack](https://github.com/hyperledger/aries-rfcs/tree/main/features/0317-please-ack) |  :x: | |
+| [0317-please-ack](https://github.com/hyperledger/aries-rfcs/tree/1.0.0-rc2/features/0317-please-ack) |  :x: | |
```

Comment:
I don't think this repo will have an `1.0.0-rc2` branch.

Reply:
found them all and fixed them.  Now I can't figure out how to push my changes into this PR.  Weird.